root/layout/index.js

```diff
@@ -88,6 +89,30 @@ const Layout = ({$c, ...props}) => (
           </p>
         </div>}
 
+      {$c.user &&
+      $c.user.birth_date &&
+      moment($c.user.birth_date).month() === moment().month() &&
+      moment($c.user.birth_date).date() === moment().date() &&
+      !getRequestCookie($c.req, 'birthday_message_dismissed_mtime') &&
```

Comment:
How often do our cookies expire? I'm assuming this would have expired before their next birthday, but I might be wrong.

Reply:
This one should probably expire after 24h?

Reply:
Should. Made changes for that.